pkg/yttlibrary/all.go

```diff
@@ -31,6 +31,7 @@ func NewAPI(replaceNodeFunc tplcore.StarlarkFunc, dataMod DataModule,
 		"json":   JSONAPI,
 		"yaml":   YAMLAPI,
 		"url":    URLAPI,
+		"net":    NetAPI,
```

Comment:
naming nit: `net` conveys networking. Something like `ip` would more properly convey that this module provides decoding, calculations, and encoding of Internet Protocol "objects" (addresses, networks/CIDRs, ... maybe interfaces...)

For example:
```
ip.parse_addr("192.0.0.1")
ip.parse_cidr("192.0.2.1/24")
```